### PR TITLE
Added rna-transcription.json test definition

### PR DESCRIPTION
**Pull request by mikeyjcat**

### CODE REVIEW

File: rna-transcription.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{
+   "#": [
+     "Language implementations vary on the issue of invalid input data.",
+     "A language may elect to simplify this task by only presenting valid",
+     "test cases.  For languages handling invalid input data as",
+     "error conditions, invalid test cases are included here and are",
+     "indicated with an expected value of null.  Note however that null is",
+     "simply an indication here in the JSON.  Actually returning null from",
```
Comment (kytrinyx) [inline rna-transcription.json]: This is great documentation for the track maintainers!

File: rna-transcription.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+     "a rna-transcription function may or may not be idiomatic in a languge.",
```
Comment (kytrinyx) [inline rna-transcription.json]: s/languge/language/

File: rna-transcription.json [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+         "rna": "A",
+         "expected": "T"
+      },
+      {
+         "description": "dna complement",
+         "rna": "UGAACCCGACAUG",
+         "expected": "ACTTGGGCTGTAC"
+      },
+      {
+         "description": "dna raises argument error",
```
Comment (kytrinyx) [inline rna-transcription.json]: Description should be about being invalid, not about raising a specific error (language-dependent implementation).

File: rna-transcription.json [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+         "dna": "U",
+         "expected": null
+      },
+      {
+         "description": "rna raises argument error",
```
Comment (kytrinyx) [inline rna-transcription.json]: ditto

File: rna-transcription.json [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+         "rna": "T",
+         "expected": null
+      },
+      {
+         "description": "dna raises argument error on completely invalid input",
```
Comment (kytrinyx) [inline rna-transcription.json]: Ditto for all the rest of the descriptions.

**Comment by mikeyjcat** [comment]
Updated as requested.

**Comment by kytrinyx** [comment]
Nice!

**Pull request merged**